eval.py

```diff
@@ -135,4 +135,4 @@ def main(args):
 
     args = parser.parse_args()
     args = DotDict(vars(args))
-    main(args)
\ No newline at end of file
+    main(args)
```

Comment:
I know newlines at the ends of files are pretty minor. But not having them occasionally results in hard-to-debug issues when scripting with typical CLI tools like `cat`, `grep`, `wc`, etc... because all of those expect lines to end with a `\n`.